libs/ui-toolkit/src/components/etherscan-link/etherscan-link.tsx

```diff
@@ -1,16 +1,15 @@
+import classNames from 'classnames';
+import type { AnchorHTMLAttributes } from 'react';
 import React from 'react';
-import type { EthereumChainId } from '../../utils/web3';
 
-const etherscanUrls: Record<EthereumChainId, string> = {
-  '0x1': 'https://etherscan.io',
-  '0x3': 'https://ropsten.etherscan.io',
-  '0x4': 'https://rinkeby.etherscan.io',
-  '0x5': 'https://goerli.etherscan.io',
-  '0x2a': 'https://kovan.etherscan.io',
+const etherscanUrls: Record<number, string> = {
+  1: 'https://etherscan.io',
+  3: 'https://ropsten.etherscan.io',
 };
```

Comment:
chore (non-blocking): should these be environment variables. General idea being the app can run against any chain without code changes, for capsule ect. However this does mean if we wanted an env switcher we would need multiple. Possibly a point for discussion, but I think our app is too ethereum environement aware atm

Reply:
Agree. I've added env vars for this for now but I also don't think its right for the ui toolkit to consume those env vars